ext/php7/ddshared.c

```diff
@@ -0,0 +1,27 @@
+#include "ddshared.h"
+
+#include "datadog/container_id.h"
+#include "datadog/string.h"
+#include "ddtrace.h"
+#include "logging.h"
+
+ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
+
+datadog_string *dd_container_id = NULL;
+
+void ddshared_minit(void) {
+    char *cgroup_file = NULL;
+    if (DDTRACE_G(test_cgroup_file) && DDTRACE_G(test_cgroup_file)[0]) {
```

Comment:
This is in `minit` -- what sets the `DDTRACE_G(test_cgroup_file)` prior to this call?

Reply:
This happens in GINIT so it's safe to access in MINIT.